pages/index.vue

```diff
@@ -663,4 +778,16 @@ export default {
 .links {
     padding-top: 15px;
 }
+
+#slack-message-block {
```

Comment:
> Instead of the "useful links"
> 
> Can we have the slack thing like at the top
> 
> ![image](https://user-images.githubusercontent.com/17459171/139131316-2fd71fde-3782-4491-bfee-3392ef5d9f4a.png)

Regarding this I changed the link to go to the anchor tag like so..
![Screen Shot 2021-10-31 at 10 25 00 PM](https://user-images.githubusercontent.com/6673572/139621593-013406bd-29c8-4a49-a045-357ab3cf22a5.png)
.